src/screens/login/OauthWebViewScreen.tsx

```diff
@@ -0,0 +1,70 @@
+import React, {useCallback, useState} from 'react';
+import {ActivityIndicator, Dimensions, StyleSheet, View, Text} from "react-native";
+import baseStyle from '../../styles/base';
+import WebView from "react-native-webview";
+import {AuthActionType, useAuthDispatch} from "../../contexts/AuthContext";
+
+
+export default function OauthWebViewScreen({route, navigation}) {
+    const {username} = route.params;
+    const authDispatch = useAuthDispatch();
+    const [loading, setLoading] = useState(true);
+
+    const onWebViewNavigate = useCallback((event) => {
+        if (event.canGoBack && event.loading) {
+            console.log("Logging in. Web View Event = ", event);
+            authDispatch({type: AuthActionType.Login, user: {username}});
+        }
+    }, [username]);
+
+    const webViewInjectedJs = `
+        document.getElementById('username').value = '${username}';
+        document.getElementById('password').value = '';
+    `;
+
+    // TODO: use oauth url here (url from backend which will redirect to google/emich login page)
+    const oauthUrl = 'https://my.emich.edu';
```

Comment:
@Mazawrath  Here is where we'll put the backend url to start the oauth process once you setup an oauth endpoint (will need a python oauth library for that)

Reply:
Got it
Should I setup a compute engine server with the server running on it so you can start testing it out?

Reply:
Yeah that'd be good to start setting up. I prefer to test with a local server instance whenever we are testing the app locally, tho.

We should actually set up a docker image for the backend, that way its easy for anyone to run the server locally and also to deploy on compute engine.